blogpost - 2021-04-28 - ready to review (#495)
SenseNet/sncom-resources: _posts/2021-04-28-hey-olvy-whats-new.md

---
title: "Hey Olvy, what's new in sensenet?"
author: [molnarm84, huszerlbence]
image: "../img/posts/olvy/olvy-whats-new.png"
tags: [saas, headless cms, releases, javascript, react, typescript, olvy]
redirect_to: https://www.sensenet.com/blog/2021-04-28-hey-olvy-whats-new
---

> These days we are getting familiar with producthunt as we want to launch our new SaaS model there. By chance we found a great tool which saved us some dev hours. Here's our experience.

Reviewer: ```suggestion
We are getting familiar with producthunt recently as we want to launch our new SaaS model there. By chance we came accross a great tool which saved us some dev hours. Here's our experience.
```

---

## Delivering release notes and updates

Since we are working on our new SaaS model we are constantly thinking how could we improve each and every part of the service. For this reason we made a [design sprint](https://www.sensenet.com/blog/2021-03-24-sncom-design-sprint) to better understand the user journey and we are currently working hard to make our demo experience more seamless.

> Content creation is also top prior so we write documentation, tutorials, blogposts and so on. But how can we deliver these new stuff to our users the most effective way?

Reviewer: ```suggestion
Content creation is also top priority, so we write documentation, tutorials, blogposts and so on. But how can we share these new stuff with our users more effectively?
```

We found a promising tool on [producthunt](https://www.producthunt.com/). The app is called [Olvy](https://olvy.co/) and it was #1 product of the day (back in 27 February) that's how we found it.

We've added it first to the [admin ui](https://docs.sensenet.com/guides/getting-started) - a central place to manage sensenet repositories in the SNaaS model - to share useful information with our users, such as release notes, new and upcoming features.

> We were so amazed by this new solution that we've included it to each surface we have.

Reviewer: ```suggestion
We were so amazed by this new solution that we've included it to each public websites and apps we have.
```

</p>

Here we would like to mention that this is not a paid promotion, we only want to write about our experience with this service. We paid full price for Olvy altough we took advantage of the early bird discount.🐣😊

The integration part was quite easy, we just had to add a small code snippet into the admin ui and tag a button with the appropriate id, that's it.

## How it works?

Olvy has a simple user interface which can be used by anyone (sales, marketing, devs) to share content in no time. You can build your post in a rich text editor, attach featured link and image to it (so it will be more appealing). Set scheduled publish and/or expiration date to make your life a bit easier.

Users see new updates as a colored badge whenewer you want (see variations above) and by clicking on the button a sidebar reveals containing all previous "releases". It is possible to get comments and reactions as feedback from your users.

<p align="center">
<img src="/img/posts/olvy/olvy-ui.png" alt="olvy user interface">
</p>

Olvy has even more features like [standalone page](https://sensenet.olvy.co/) for releases and more to discover in the future.

> For us this is the ideal solution to share info about our new releases like new features, fixed bugs and so on (both frontend and backend). As mentioned above, you can still reach our releases on our website and GitHub as well, but now you'll be notified about any news around sensenet immediately once you visit our public websites or apps.

Reviewer: ```suggestion
For us this is the ideal solution to share info about our new releases like new features, fixed bugs and so on (both frontend and backend). As mentioned above, you can still reach our releases [on our website](https://www.sensenet.com/backend-updates) and [GitHub](https://github.com/SenseNet/sn-client/releases) as well, but now you'll be notified about any news around sensenet immediately once you visit our public websites or apps.
```
